### ISSUE TITLE
unable to run pytest

### ISSUE THREAD

**Issue by SlowMo24** [GIScience/ohsome-py#56]
The contribution guidelines are a bit hard to follow for newbies to pytest etc. (especially if using conda). Once I got it running the following error occured:

```
ERROR: usage: pytest [options] [file_or_dir] [file_or_dir] [...]
pytest: error: unrecognized arguments: --cov=ohsome --cov-report=xml --random-order
  inifile: .../ohsome-py/tox.ini
  rootdir: .../ohsome-py
```

**Comment by SlowMo24** [comment]
so you need to have `pytest-random-order` and `pytest-cov` installed. Mabe mention that in the README?

**Comment by redfrexx** [comment]
I've added the dependencies for testing in https://github.com/GIScience/ohsome-py/commit/50f1d2eb696f04b887e360e3c8fab59237c75e5a.
